source/zipkin/span_buffer.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <vector>
```

Comment:
Remove all standard headers that are already in precompiled.h (I won't comment on this in other places).